src/SelfServe/SqlX/SqlxTypes.ts

```diff
@@ -0,0 +1,31 @@
+export type SqlxServiceResource = {
+  id: string;
+  name: string;
+  type: string;
+  properties: SqlxServiceProps;
+  locations: SqlxServiceLocations;
+};
+export type SqlxServiceProps = {
+  serviceType: string;
+  creationTime: string;
+  status: string;
+  instanceSize: string;
+  instanceCount: number;
+  sqlxEndPoint: string;
+};
+
+export type SqlxServiceLocations = {
+  location: string;
+  status: string;
+  sqlxEndpoint: string;
+};
+
+export type UpdateDedicatedGatewayRequestParameters = {
```

Comment:
>UpdateDedicatedGatewayRequestParameters [](start = 12, length = 39)

do you see more coming in this? if not, why not directly use UpdateDedicatedGatewayRequestProperties

Reply:
Yes, we do see a lot more updates to "Update RP API", also I want to follow a pattern which is consistent with the other RP calls.